src/disrupt.sh

```diff
@@ -23,15 +23,25 @@ if [[ ! -d "${IMG_ASSET_DIR}" || ! $(find "${IMG_ASSET_DIR}" -mindepth 1 -iname
     DICE_ROLL=0
 fi
 
-# add the image if necessary
-if [ "${DICE_ROLL}" -eq 1 ]; then
-    RANDOM_IMAGE=$(find "${IMG_ASSET_DIR}" -type f | shuf -n 1)
+#Check for an image associated with the fact
```

Comment:
Please add a space between the start of the comment to be consistent with the rest of the codebase.